### DISCUSSION TITLE
Trying to open gre over ipsec ! Whear is mistake ?

### DISCUSSION THREAD

**Discussion by killgapwalker** [strongswan/strongswan#367] [General]
ip tunnel add ipsec0 local (my private address to second laptop)-192.168.43.221 remote (nordvpn ip)-196.196.196.10 mode gre
ip link set ipsec0 up
ip route add 10.1.0.0/16 dev ipsec0

Im added SPI's in IP route address which give me strongswan is that correct ?

ip -s tunnel show ipsec0
ipsec0: gre/ip remote ******* local ******** ttl inherit
RX: Packets    Bytes        Errors CsumErrs OutOfSeq Mcasts
    0          0            0      0        0        0       
TX: Packets    Bytes        Errors DeadLoop NoRoute  NoBufs
    0          0            0      0        0        0

**Comment by killgapwalker** [comment]
Dhclient doesnt help.

**Reply by Thermi** [reply]
Of course not. The other side probably doesn't have it configured. And GRE does not inherently have anything to do with IPsec.
The only connection between IPsec and GRE at all is only possibly done when you put it through a VPN tunnel by routing it through there. That's not a parameter of the GRE interface though but by the negotiated IPsec policies and the routing engine on your host.

**Reply by killgapwalker** [reply]
Do you know gre hosts? Is it real with your client https://m.habr.com/ru/company/xakep/blog/256659/ ??

**Reply by Thermi** [reply]
What is that supposed to mean "gre hosts"?

**Reply by killgapwalker** [reply]
preconfigured other side, man... or gre is for lan ?

**Reply by killgapwalker** [reply]
I have problem with ipsec , this traffic doesn't encapsulate in wlp interface and going to other hosts without route...

**Reply by Thermi** [reply]
Okay. Then please provide all information as listed in #196
